### DISCUSSION TITLE
[help] Can I load a targets object outside of a targets repo?

### DISCUSSION THREAD

**Discussion by bmaitner** [ropensci/targets#1097] [Help] [answered]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Hi all, apologies if this is a silly question.

Is it possible to load a targets object from outside of a targets directory?  For example, can I take a file from the _targets/objects folder of directory A, copy it to a non-targets directory B, and load it outside of the targets environment it was created in? 

I understand that in most contexts, the better solution would probably be to have targets the object simply be a pointer to a file that can be read in elsewhere, but I'm interested in exporting large target objects that are automatically generated by stantargets and would prefer not to have multiple versions of these if I can avoid it. It seems like in principle this should be easy, but I've dug through the documentation and haven't found anything (although I may have just missed it or used the wrong keywords).

Thanks in advance for any help!

**Comment by gorkang** [comment, marked as answer]
They are just rds objects, so `readRDS()` or similar should work.

**Reply by bmaitner** [reply]
That's what I had thought, but I seem to be having issues.  Upon further inspection, the issue is only with files generated by stantargets, so I think this question is more appropriate over there.  Apologies for the trouble!

**Reply by wlandau** [reply]
`tar_read()` has a `store` argument which lets you specify the path to the `_targets/` data store. You could alternatively make the pipeline write external files, i.e. `format = "file"`: https://books.ropensci.org/targets/data.html#external-files

**Reply by asadow** [reply]
I use `tar_read()` and have no problems.
